src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -123,6 +123,16 @@ const isTitleEllipsized = (node) => {
 
     return textHeight > nodeHeight;
 };
+const getContainer = (tooltipNode) => {
+    const { tooltipUseModalContainer, tooltipContainerSelector } = tooltipNode.dataset;
```

Comment:
I'm not sure if I like the introduction of new data attr "tooltip-use-modal-container", why we cannot use `data-tooltip-container-selector=".ibexa-modal"`?